paper/src/main/resources/plugin.yml

```diff
@@ -1,10 +1,10 @@
 name: ${name}
 main: com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks
 author: ProgrammerDan
-authors: [ProgrammerDan, Maxopoly]
+authors: [ ProgrammerDan, Maxopoly ]
```

Comment:
Why?

Reply:
Autoformat I guess? I don't remember formatting that file